test/versioned/grpc/grpc-server.js

```diff
@@ -11,12 +11,16 @@ module.exports = function createServerMethods(server) {
       // add the metadata from client that the server receives so we can assert DT functionality
       server.metadataMap.set(name, metadata.internalRepr)
       const message = `Hello ${name}`
-      cb(null, { message })
+      // Call setTimeout so we create a new child segment to test on
+      setTimeout(() => {
+        cb(null, { message })
+      }, 0)
     },
     sayHelloClientStream: function sayHelloCStream(call, cb) {
       const { metadata } = call
       const names = []
       call.on('data', function (clientStream) {
+        // TODO: Try creating a segment here, look at test code on how to insert a segment here
```

Comment:
Yes, some sort of simulation of work to make sure the spans would show up in the transaction. But now after talking with Michael I am pretty sure they would not. This could be a big omission of the instrumentation as it currently stands, no spans at all for things that happen inside these event handlers, which I think are outside the transaction we created for the server instrumentation.

Might be okay to just release what I have so far and later come to revisit this omission.

Oh, I also want to test the extra segment on the unary call created above with `setTimeout`, just to make the test a little more interesting and make sure segments are there.

Reply:
I don't mind we leave a todo but if we don't have a ticket to do the work we'll prob never address.

Reply:
Created [the ticket](https://issues.newrelic.com/browse/NEWRELIC-1460), have removed the TODO from here.